Lib/fontbakery/specifications/ufo_source_test.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
```

Comment:
could you please rename this file so that its name is identical to the name of the script it tests, except for the `_test` suffix?

I mean  to call it `ufo_sources_test.py`instead of `ufo_source_test.py` in order to match **Lib/fontbakery/specifications/ufo_sources.py**